_haikus/haikuchris.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: haiku
+title: my firs time using git
+author: chris
+---
+
+today's History
+tommorow is progress
```

Comment:
this line should be seven syllables

Reply:
thank you

Reply:
How do i upload my updates i'm having trouble, because this my first time using git and github

Reply:
You should be able to modify your file locally and just update this pull request with a new commit. If it is easier, you can close this pull request and open a new one.